web/adventures/test_adventures.py

```diff
@@ -57,3 +58,33 @@ def test_URLListField_get_prep_value(self):
         """Test URLListField's to_python with none"""
         url_list = URLListField()
         self.assertEqual(url_list.get_prep_value(["www.google.com", "another.website.io"]), "www.google.com,another.website.io")
+
+
+class AdventureByIdTestCase(TestCase):
+
+    def setUp(self):
+        self.client = Client()
+
+    def test_adventure_by_id_get_success(self):
+        test_adv = Adventure.objects.create(name="LMoP", links=["www.google.com", "another.website.io"])
```

Comment:
You can initialize the model with a dictionary in which keys match field names with:
`new_mymodel = MyModel.objects.create(**data_dict)`
This would ensure that the model is properly initialized with your correct input and make it DRY.